docs/migration/01_migration_plan.md

```
@@ -0,0 +1,838 @@
+| Presentation pages | ~39 directories |
+| Pilot module | Invitation |
+
+This is not a refactoring. It is a progressive rewrite of the app's backbone.
+
+---
+
+## 2. Current architecture vs target architecture
+
+### 2.1 Current architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)